lib/runtime/wasmer/imports.go

```diff
@@ -318,10 +322,31 @@ func ext_misc_print_utf8_version_1(context unsafe.Pointer, data C.int64_t) {
 }
 
 //export ext_misc_runtime_version_version_1
-func ext_misc_runtime_version_version_1(context unsafe.Pointer, z C.int64_t) C.int64_t {
+func ext_misc_runtime_version_version_1(context unsafe.Pointer, dataSpan C.int64_t) C.int64_t {
```

Comment:
according to the spec the input data is another wasm blob.... so this means you will need to instantiate a runtime inside this function using the input data, call `instance.Version` on it, then return that data

Reply:
I have modified it. But what should I pass in config when creating a new instance?

Reply:
you can pass the existing `Storage` in the context, and `ImportsNodeRuntime` as the imports. the rest you can leave out, since the `Version` call won't be using them